### PR TITLE
fix: change wrong links in PluginHub

### LINKED ISSUE #462: page: failed to load the ico of plugin
uri: https://apisix.apache.org/pluginhub/

image

### PR DESCRIPTION
**Pull request by Baoyuantop**
Fixes: #462 

Screenshots of the change:

image

### CODE REVIEW

**Comment by Baoyuantop** [comment]
@liuxiran

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -463,7 +463,7 @@ module.exports = {
           position: "right",
           items: [
             {
-              to: "/hub",
+              to: "/pluginhub",
```
Comment (juzhiyuan) [inline website/docusaurus.config.js]:
```suggestion
              to: "/plugins",
```

Comment (bzp2010) [reply website/docusaurus.config.js]: +1 agree with this

Comment (liuxiran) [reply website/docusaurus.config.js]: +1 agree

**Review by liuxiran — APPROVED** [review]
LGTM, thanks @Baoyuantop

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**